src/legacy/core_plugins/console/public/index.html

```diff
@@ -1,4 +1,7 @@
-<kbn-top-nav name="console" config="topNavMenu"></kbn-top-nav>
+<kbn-top-nav2 
```

Comment:
`kbn-top-nav2`? 😸

Reply:
Yeah, the goal is to kill the old directive and then to rename this one back to be kbn-top-nav :)
This is a temporary solution that will not be there when V7.4 is ready.

Reply:
Changed to `kbn-top-nav-v2` :)